app/actions/TrezorActions.js

```diff
@@ -993,6 +998,46 @@ export const getWalletCreationMasterPubKey = () => async (
   }
 };
 
+export const TRZ_VOTINGXPRIVSEED_ATTEMPT = "TRZ_VOTINGXPRIVSEED_ATTEMPT";
+export const TRZ_VOTINGXPRIVSEED_FAILED = "TRZ_VOTINGXPRIVSEED_FAILED";
+export const TRZ_VOTINGXPRIVSEED_SUCCESS = "TRZ_VOTINGXPRIVSEED_SUCCESS";
+
+/* eslint-disable no-unused-vars */
+const votingXprivSeed = () => async (dispatch, getState) => {
+  /* eslint-enable no-unused-vars */
+  dispatch({ type: TRZ_VOTINGXPRIVSEED_ATTEMPT });
+
+  if (noDevice(getState)) {
+    dispatch({
+      error: "Device not connected",
+      type: TRZ_VOTINGXPRIVSEED_FAILED
+    });
+    return;
+  }
+
+  const chainParams = selectors.chainParams(getState());
+
+  try {
+    const ckv = await deviceRun(dispatch, getState, async () => {
+      const res = await session.cipherKeyValue({
+        path: accountPath(WALLET_ACCOUNT, chainParams.HDCoinType),
```

Comment:
Probably best to use `addressPath` instead of simply `accountPath`

Reply:
Fine with me, but why is that?

Reply:
We can extract private keys of individual addresses using standard CLI tools if needed.

Reply:
By cli tools do you mean [trezor-cli](https://wiki.trezor.io/Using_trezorctl_commands_with_Trezor)? We cannot extract any private keys afaik.

Reply:
Standard decred tools if you have the seed imported in a dcrwallet instance

Reply:
oh, that's right, the trezor seed to dcrwallet, I understand now.